pages/osx/date.md

```diff
@@ -0,0 +1,19 @@
+# date
+
+> Set or display the system date.
+
+- Display the date using the default locale:
+
+`date +"%c"`
+
+- Display the date in UTC and ISO 8601 format:
+
+`date -u +"%Y-%m-%dT%H:%M:%SZ"`
+
+- Display the number of seconds since the unix epoch:
+
+`date +%s`
+
+- Convert a unix expoch timestamp to a date:
```

Comment:
Neither `j` nor `r` are particularly illuminating flags in the man page:

<img width="787" alt="screenshot of iterm2 8-09-16 9-55-54 pm" src="https://cloud.githubusercontent.com/assets/811954/18345318/0e965e1c-760f-11e6-94d6-ca83ceabbb65.png">